### PR TITLE
feat: add en book page

### PR DESCRIPTION
**Pull request by davidChan3000**

### CODE REVIEW

**Review by menghaoranss — CHANGES_REQUESTED** [review]
Can the picture book.png be compressed?

File: style/bookDesc.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+#bookDesc .learn-con ul{
+    margin-bottom: 40px;
+}
+
+#bookDesc .learn-con li{
+    font-size: 22px;
+    margin-bottom: 25px;
+}
```
Comment (RaigorJiang) [inline style/bookDesc.css]: Blank line missed here.

**Comment by davidChan3000** [comment]
> Can the picture book.png be compressed?

compressed

File: style/bookDesc.css (hunk 1 above)
Comment (davidChan3000) [reply style/bookDesc.css]: added

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**